File: rb/test/exists_test.rb
# AppStoreMetadata SDK exists test

require "minitest/autorun"
require_relative "../AppStoreMetadata_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = AppStoreMetadataSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- AppStoreMetadata SDK exists test

local sdk = require("app-store-metadata_sdk")

describe("AppStoreMetadataSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
